app/src/main/res/menu/menu_main.xml

```diff
@@ -1,10 +0,0 @@
-<menu xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
This file is removed already. It's strange how GitHub shows it. It looks like it's an empty file, but it's totally gone